test/ViewModel/Converter/PromotionalCollectionContentHeaderConverterTest.php

```diff
@@ -0,0 +1,36 @@
+<?php
```

Comment:
https://alfred.elifesciences.org/blue/organizations/jenkins/pull-requests-projects%2Fjournal/detail/PR-1257/10/tests confirms the generated tests inherited from `ModelConverterTestCase` are running fine